ee/clickhouse/queries/funnels/base.py

```diff
@@ -311,3 +309,40 @@ def _get_step_time_avgs(self, max_steps: int):
     @abstractmethod
     def get_query(self, format_properties):
         pass
+
+    def _get_breakdown_select_prop(self) -> str:
+        if self._filter.breakdown:
+            self.params.update({"breakdown": self._filter.breakdown})
+            if self._filter.breakdown_type == "person":
+                return f", JSONExtractRaw(person_props, %(breakdown)s) as prop"
+            elif self._filter.breakdown_type == "event":
+                return f", JSONExtractRaw(properties, %(breakdown)s) as prop"
+
+        return ""
+
+    def _get_breakdown_conditions(self) -> str:
+        if self._filter.breakdown:
+            limit = 5
+            first_entity = next(x for x in self._filter.entities if x.order == 0)
+            if not first_entity:
+                ValidationError("An entity with order 0 was not provided")
+            values = []
+            if self._filter.breakdown_type == "person":
+                values = BreakdownProps(
+                    self._filter, first_entity, "count(*)", self._team.pk, limit=limit
+                ).get_person_prop_values()
+            elif self._filter.breakdown_type == "event":
+                values = BreakdownProps(
+                    self._filter, first_entity, "count(*)", self._team.pk, limit=limit
+                ).get_event_prop_values()
+            self.params.update({"breakdown_values": values})
+
+            return "prop IN %(breakdown_values)s"
+        else:
+            return ""
+
+    def _get_breakdown_prop(self) -> str:
```

Comment:
small nit / not sure if worth tackling here since other discrepancies remain: we decided not to return query strings with a comma in the beginning.

Reply:
Will save that change for a later update where we adjust how the handling is done in all this query formatting